Client/Application/LeanplumIntegration.swift

```diff
@@ -116,7 +121,15 @@ class LeanplumIntegration {
 
         Leanplum.start(userAttributes: userAttributesDict)
 
-        Leanplum.track(LeanplumEventName.openedApp.rawValue)
+        track(eventName: LeanplumEventName.openedApp)
```

Comment:
yes it does; but it's cleaner code since the rest of the app uses track() so might as well use it here